docs/self-hosted/kubernetes-maintenance.md

```diff
@@ -31,3 +31,10 @@ kubectl kots admin-console upgrade -n NAMESPACE
 
 Replace `NAMESPACE` with the namespace in your cluster where KOTS is installed.
 
+### Updating or Replacing the SSL Certificate
+
+The following applies only to kURL installations. To update or replace the certificate, you must first restore the ability 
```

Comment:
Coordinate with https://github.com/packagist/private-packagist-docs/pull/106 about where to add this note

Reply:
@stevenrombauts I think it makes sense that we rather use your general notice on top of the page instead of a notice just for this section. I would remove it here if that's ok with you.